### PR TITLE
initial checkin of formatting guide

### PR DESCRIPTION
**Pull request by marcacohen**

### CODE REVIEW

File: FORMAT-GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+    It is recommended that you keep your infoboxes clean, concise and focused on a single topic. If you have information which may be useful to know but is not a critical part of the codelab instructions, then you should link to that information from the FAQ section rather than including it as an infobox.
+
+1. Command-line Snippets
+
+    It's often the case that you will have command-line instructions or log messages which are formatted with a monospaced font and have pre-defined whitespace. You can add these sections to your codelab by creating a **single-cell table** and making sure that all the text is formatted using the **Consolas font**.
+
+1. Code Snippets
+
+    Code snippets with syntax highlighting can be added by creating a **single-cell table** and making sure that all the text is formatted using the **<code>Courier New</code></strong> font.
```
Comment (nicolasgarnier) [inline FORMAT-GUIDE.md]: ` **<code>Courier New</code></strong>` isn't working. Maybe: ```**`Courier New`**```

File: FORMAT-GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+    You can make it really easy to get started by including direct download buttons inline in your codelab. In order to add a button to your codelab, simply add a hyperlink and make sure that it is highlighted with a **dark green 1** background.
+
+    Additionally, if the link text begins with the word "Download", a file download icon will be added to the button.
+
+1. Per-step Time Estimates
+
+    Many participants are not fully committed to completing a codelab when they start it. One of the ways that we can keep them in our codelab is by giving them accurate estimates about how much additional effort is required to complete the codelab at each step.
+
+    In order to add this feature to your codelab, simply annotate each step in your codelab doc with a Duration: which uses **dark grey 1 **text.
```
Comment (nicolasgarnier) [inline FORMAT-GUIDE.md]: `**dark grey 1 **text.` => `**dark grey 1** text.`

File: FORMAT-GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+    That's it. The codelab framework will do everything else for you. If you forget to annotate a step with a duration, the default is 1:00. Also, if the last step of your codelab is just a congratulations page, you should set the duration of that step to **0**.
+
+1. Conditional Steps
+
+    Sometimes it's useful to have different versions of a codelab for different environments. For example, you might have some steps that only apply to students who take the codelab in a classroom setting while other steps only apply to people who are following the instructions at their own pace online.
+
+    The format is similar to the duration metadata. You simply specify one or more environments with an Environment: which uses **dark grey 1 **text. The Duration and/or Environment fields, when present, should be followed by a blank line and should be set in normal text (not in Heading 1, lest they be considered part of the step title).
```
Comment (nicolasgarnier) [inline FORMAT-GUIDE.md]: `**dark grey 1 **text.` => `**dark grey 1** text.`

File: FORMAT-GUIDE.md (hunk 1 above)
Comment (marcacohen) [reply FORMAT-GUIDE.md]: fixed

File: FORMAT-GUIDE.md (hunk 2 above)
Comment (marcacohen) [reply FORMAT-GUIDE.md]: fixed

File: FORMAT-GUIDE.md (hunk 3 above)
Comment (marcacohen) [reply FORMAT-GUIDE.md]: fixed

**Comment by marcacohen** [comment]
thanks for review @nicolasgarnier , PTAL

**Pull request merged**